photon-ml/src/integTest/scala/com/linkedin/photon/ml/data/AvroDataReaderTest.scala

```diff
@@ -87,7 +89,7 @@ class AvroDataReaderTest extends SparkTestUtils {
 
     } catch {
       case se: SparkException => assertTrue(se.getMessage.contains("Duplicate features found"))
-      case e: Exception => throw(e)
+      case e: Exception => throw e
```

Comment:
The previous way is preferred by the Databricks Style guide: https://github.com/databricks/scala-style-guide#infix